packages/components/organisms/f-header/src/components/Logo.vue

```diff
@@ -46,33 +48,33 @@ export default {
         headerBackgroundTheme: {
             type: String,
             default: 'white'
+        },
+        isOpen: {
```

Comment:
Super tiny one – I'd call this `isNavOpen` (assuming that's what it refers to?) as looks like there's a few references to different open states across the component.